### PR TITLE
docs(en-US): polish iOS integration guide

### PR DESCRIPTION
**Pull request by gao-sun**
polish iOS integration guide, ready for prod

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/ios.mdx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+sidebar_label: iOS
+---
+
+TBD
```
Comment (demonzoo) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/integrate-logto/ios.mdx:5]: nit: Can we decide the label now?

**Review by demonzoo — APPROVED** [review]

File: docs/docs/recipes/integrate-logto/components/AppNote.tsx [hunk 2]
```diff
@@ -8,8 +8,10 @@ type Props = {
 const AppNote = ({ type }: Props) => {
   return (
     <Admonition type="note">
-      This tutorial assumes you have created a {type} Application in Admin Console. If you are not
-      ready, read this one before continuing.
+      This tutorial assumes you have created an Application of type &quot;{type}&quot; in Admin
+      Console. If you are not ready,{' '}
+      <a href="../../tutorials/get-started/create-and-integrate-the-first-app">read this</a> one
```
Comment (darcyYe) [inline docs/docs/recipes/integrate-logto/components/AppNote.tsx:13]: The link should be mounted to `read this one` or `this one`? Or maybe you can simply remove `one`.

File: docs/docs/recipes/integrate-logto/ios.mdx [hunk 3]
```diff
@@ -61,11 +62,27 @@ let config = try? LogtoConfig(
...
+
+[Redirect URI](https://www.oauth.com/oauth2-servers/redirect-uris/) is an OAuth 2.0 concept which implies the location should redirect after authentication.
+
+:::info
+The Redirect URI in iOS SDK is only for internal use. There's _NO NEED_ to add a [Custom URL Scheme](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) until a connector asks.
+:::
+
+### Sign in with browser
+
 <SignInNote calling=".signInWithBrowser(redirectUri:)" />
```
Comment (darcyYe) [inline docs/docs/recipes/integrate-logto/ios.mdx:79]:
```suggestion
<SignInNote calling=".signInWithBrowser(redirectUri)" />
```

**Review by darcyYe — APPROVED** [review]
Overall LGTM

File: docs/docs/recipes/integrate-logto/components/AppNote.tsx (hunk 2 above)
Comment (gao-sun) [reply docs/docs/recipes/integrate-logto/components/AppNote.tsx:13]: yea I refactored in another PR

File: docs/docs/recipes/integrate-logto/ios.mdx (hunk 3 above)
Comment (gao-sun) [reply docs/docs/recipes/integrate-logto/ios.mdx:79]: This is by design. An iOS convention

**Pull request merged**